content/_partials/mailing-lists/jenkinsci-board.adoc

```diff
@@ -1,3 +1,3 @@
 Mailing list to contact https://www.jenkins.io/project/board/[Jenkins board members].
 
-NOTE: the subscription to this mailing list is reserved to Jenkins board members.
+NOTE: The subscription to this mailing list is reserved to Jenkins board members. Do not message this list for support with your Jenkins instance or plugins.
```

Comment:
I'm not a native speaker but message somehow sounds wrong, does "use" make more sense?

Reply:
```suggestion
NOTE: The subscription to this mailing list is reserved to Jenkins board members. Do not send messages to this list for support with your Jenkins instance or plugins.
```
"Use" definitely makes sense here, otherwise my suggestion would be to add a couple words to have the original wording be as clear as possible.